books/std/util/defines.lisp

```diff
@@ -733,6 +768,12 @@ encapsulate), and is mainly meant as a tool for macro developers.</dd>
        ((unless (consp (cdr gutslist)))
         (raise "Error in ~x0: expected more than one function." name))
 
+       (verify-guards-after-returns
+         (eq (cdr (assoc :verify-guards kwd-alist)) :after-returns))
+       (kwd-alist (if verify-guards-after-returns
+                      (put-assoc :verify-guards 'nil kwd-alist)
```

Comment:
I'd prefer that the kwd-alist stored in the defines-guts isn't changed like this -- if defines was called with :verify-guards :after-returns then it should reflect that

Reply:
Sure, I can avoid modifying `kwd-alist` and just do the `put-assoc` when calling `get-xargs-from-kwd-alist`.